app/controllers/api/v1/briefcase_controller.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    class BriefcaseController < ApplicationController
+      before_action :authenticate_api_v1_user!
+
+      def create
+        briefcase = Briefcase.new(
+          expiring_at: Time.now.utc + 604800,
+          user_id: current_api_v1_user[:id]
+        )
+        if briefcase.save
+          render json: briefcase, status: 201
+        else
+          render json: { error: 'Bad request ( invalid data )' }, status: 400
+        end
+      rescue StandardError
+        render json: { error: 'Bad request ( invalid data )' }, status: 400
+      end
+
+      def index
+        briefcases = get_briefcases_for_current_user
+        render json: briefcases, status: 200
+      end
+
+      def update
+        briefcase = get_briefcase_by_id
+        if briefcase and get_stock
+          stock = get_stock
```

Comment:
Ходить три раза в базу - плохо :)
```suggestion
        briefcase = get_briefcase_by_id
        stock = get_stock
        if briefcase and stock
```